std/path.d

```diff
@@ -3947,7 +3947,7 @@ string expandTilde(string inputPath) nothrow
             string cp;
             if (char_pos < path.length)
                 // Append something from path
-                cp = cast(string)(c_path[0 .. end] ~ path[char_pos .. $]);
+                cp = text(c_path[0 .. end], path[char_pos .. $]);
```

Comment:
IIRC `a ~ b` is faster than `text` because for the former the exact length is know and thus only one allocation happens whereas text uses an appender.
(I doubt that it matters here though)